Userland/Utilities/init.cpp

```diff
@@ -124,7 +124,7 @@ static ErrorOr<void> prepare_bare_minimum_devtmpfs_directory_structure()
     TRY(populate_device_node_with_symlink(DeviceNodeType::Character, "/dev/mem"sv, 0600, 1, 1));
```

Comment:
Nitpick: this commit title should be "Userland/init: Change file permissions for /dev/random device node"